### ISSUE TITLE
Launcher form builder provides no help text to screen readers

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#5260]
When using the launcher form builder, we need to make sure it is clear that you are in a form builder context, and not filling out the actual form. For example, all `select` widgets should have help text along the lines of `select default value for OPTION`, as well as better help text on the various buttons (like add, remove, edit) to make it clear what each of those are doing.
